src/components/recipe-form/RecipeForm.tsx

```diff
@@ -0,0 +1,207 @@
+"use client";
+
+import { useFieldArray, useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Plus, Trash2 } from "lucide-react";
```

Comment:
Norbert added Plus as a icon from hero-icons, we should discuss the source of icons and refactor whole code probably (on the call and I will create a task for this)